ocropus-econf

```diff
@@ -84,14 +84,14 @@ for fname,e,t,m,c in outputs:
 if perfile is not None: perfile.close()
 if allconf is not None: allconf.close()
 
-print "errors    %8d"%errs
-print "missing   %8d"%missing
-print "total     %8d"%total
-print "err       %8.3f %%"%(errs*100.0/total)
-print "errnomiss %8.3f %%"%((errs-missing)*100.0/total)
+print("errors    %8d"%errs)
```

Comment:
Is it also needed if print only gets one argument? At least for Python 2.7.9 it works without that statement.